pages/administratorAdmins/administratorAdmins.html.js

```diff
@@ -54,7 +54,7 @@ function AdministratorAdmins({ admins, resLocals }) {
                   <tr>
                     <th>UID</th>
                     <th>Name</th>
-                    <th></th>
+                    <th>Actions</th>
```

Comment:
I could decide exactly what to do in all these cases. In some headings (like here), I just added a generic "Actions" title; in others, I still added that title but made it only visible to screen readers. We should pick one and be consistent; I lean towards always making these titles visible.

In the long-term, it'd be better on many pages to not use a table at all. That's outside the scope of this PR though.

Reply:
I would vote for "Actions" title to be visible everywhere, I can't see any harm in it.

I agree that changing from table layouts is beyond the scope of this PR.

Reply:
I'm on board with making it visible, at least until we can come up with something better than the table layout.